README.md

```diff
@@ -30,7 +30,28 @@ In-depth docs are available [on the Terraform registry](https://registry.terrafo
 
 ## Development
 
-To do development you need Go installed, this repo cloned and that's about it. It has not been tested on Windows, so if you find problems let us know.
+If you're new to Go:
+
+1. Install Go `brew install golang`
+2. Make a Go development directory wherever you like `mkdir go_projects`
+3. Add the following config to your profile
+```
+export GOPATH=$HOME/../go_projects # edit with your go_projects dir
+export GOROOT=/usr/local/opt/go/libexec
+export PATH=$PATH:$GOPATH/bin
+export PATH=$PATH:$GOROOT/bin
```

Comment:
Are these still necessary with Go 1.13+?

Reply:
I literally just started with Go so you'll know more than me! I can try on a fresh environment without

Reply:
Answer is it depends how you install. If you install Go with the package installer from [here](https://golang.org/doc/install) then the required directories are added to PATH. If you install with brew, you need to do it yourself.

Reply:
You're right, GOROOT no longer required. I've tidied it up and also added a missing step for `goimport`.